model/result/MessagesService.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace oat\taoLtiConsumer\model\result;
+
+use oat\taoLtiConsumer\model\result\XmlFormatterService;
+
+class MessagesService
+{
+    const SERVICE_ID = 'result_service';
+    const FAILURE_MESSAGE = 'failure';
+    const SUCCESS_MESSAGE = 'success';
+
+    const STATUS_INVALID_SCORE = 400;
+    const STATUS_DELIVERY_EXECUTION_NOT_FOUND = 404;
+    const STATUS_METHOD_NOT_IMPLEMENTED = 501;
+    const STATUS_INTERNAL_SERVER_ERROR = 500;
+    const STATUS_SUCCESS = 201;
+
+    public static $statuses = array(
+        self::STATUS_INVALID_SCORE => 'Invalid score',
+        self::STATUS_DELIVERY_EXECUTION_NOT_FOUND => 'DeliveryExecution not found',
+        self::STATUS_METHOD_NOT_IMPLEMENTED => 'Method not implemented',
+        self::STATUS_INTERNAL_SERVER_ERROR => 'Internal server error, please retry',
+    );
+
+    public static function buildMessageData($code, $result)
```

Comment:
Why making this method static?

Reply:
Let's keep it as is. TODO is added for changing it to non-static type